media/scss/bootstrap-4.6.2/_nav.scss

```diff
@@ -14,7 +14,7 @@
 .nav-link {
   display: block;
   padding: $nav-link-padding-y $nav-link-padding-x;
-  text-decoration: if($link-decoration == none, null, none);
+  text-decoration: if($link-decoration ==none, null, none);
```

Comment:
These are the bootstrap source files, so we shouldn't be editing these. Any changes here will be over-written when we update bootstrap.

Reply:
Right. I opened it up and probably hit save. Looks like the VS formatter moved things around.